notes/snap-waived-interview.vbs

```diff
@@ -271,7 +288,7 @@ Function needed_info_dialog(needed_info_array)
 		If previous_verif_found = True Then PushButton 20, 350, 125, 15, "View Previous Verifications", view_previous_verifs_btn
 	End If
 	PushButton 345, 350, 75, 15, "Complete Contact", contact_completed
-	If last_page <> page_display Then PushButton 290, 350, 50, 15, "Next", next_btn
+	If page_display <> q_page_xtra Then PushButton 290, 350, 50, 15, "Next", next_btn
```

Comment:
Verification buttons on the xtra dialog do not provide a dialog to enter information. Seems like it's looping back to the xtra dialog without providing verf dialog.